### PR TITLE
skip base model for compatibility

### PR DESCRIPTION
**Pull request by guochaorong**
skip runnning for  base models whose dirs have no run.xsh

### CODE REVIEW

File: eva.xsh [hunk 1]
```diff
@@ -49,6 +49,9 @@ def main():
     tasks = get_changed_tasks(args)
     times = args.times
     for task in tasks:
+        run_file = os.path.join(config.baseline_path, '%s/run.xsh' % task)
+        if not os.path.exists(run_file):
+            continue
```
Comment (Superjomn) [inline eva.xsh]: Add a warning here, "skip xxx directory, it is not a task"

File: eva.xsh [hunk 2]
```diff
@@ -75,7 +78,7 @@ def run_task(task_name, times):
     cd @(config.workspace)
     env = {}
     exec('from tasks.%s.continuous_evaluation import tracking_kpis'
-             % task_name, env)
+             % task_name.split('-')[0], env)
```
Comment (Superjomn) [inline eva.xsh]: why `-` here?

File: eva.xsh (hunk 1 above)
Comment (guochaorong) [reply eva.xsh]: ok~

File: eva.xsh (hunk 2 above)
Comment (guochaorong) [reply eva.xsh]: import kpis from base model  
base model is as  resnet50   
models are like resnet50-v100-cardn  which are splited by '-'

File: eva.xsh [hunk 3]
```diff
@@ -79,7 +79,7 @@ def run_task(task_name, times):
     cd @(config.workspace)
     env = {}
     exec('from tasks.%s.continuous_evaluation import tracking_kpis'
-             % task_name.split('-')[0], env)
```
Comment (guochaorong) [inline eva.xsh]: 每个模型用自己的kpi变量脚本， 也可以软链到base模型kpi变量

File: eva.xsh [hunk 4]
```diff
@@ -49,6 +49,10 @@ def main():
     tasks = get_changed_tasks(args)
     times = args.times
     for task in tasks:
+        # skip base task directory, it is not a real task
```
Comment (Superjomn) [inline eva.xsh:52]: 还有base task 吗？

Comment (guochaorong) [reply eva.xsh:52]: 嗯啊， eva.xsh 也一样， base task中没有启动脚本， 不跑~

**Comment by guochaorong** [comment]
take Mr chun‘s suggestion， treat all models as the same~

**Pull request closed without merging**